snc-library.sh

```diff
@@ -140,35 +140,15 @@ function create_json_description {
 }
 
 
-function create_pvs() {
-    local bundle_type=$1
-
-    # Create hostpath-provisioner namespace
-    retry ${OC} apply -f kubevirt-hostpath-provisioner-csi/namespace.yaml
-    # Add external provisioner RBACs
-    retry ${OC} apply -f kubevirt-hostpath-provisioner-csi/external-provisioner-rbac.yaml -n hostpath-provisioner
-    # Create CSIDriver/kubevirt.io.hostpath-provisioner resource
-    retry ${OC} apply -f kubevirt-hostpath-provisioner-csi/csi-driver-hostpath-provisioner.yaml -n hostpath-provisioner
-    # Apply SCC allowin hostpath-provisioner containers to run as root and access host network
-    retry ${OC} apply -f kubevirt-hostpath-provisioner-csi/kubevirt-hostpath-security-constraints-csi.yaml
-
-    # Deploy csi driver components
-    if [[ ${bundle_type} == "snc" ]]; then
-        # in case of OCP we want the images to come from registry.redhat.io
-        # this is done using the kustomize.yaml file
-        retry ${OC} apply -k kubevirt-hostpath-provisioner-csi/csi-driver -n hostpath-provisioner
-    else
-        retry ${OC} apply -f kubevirt-hostpath-provisioner-csi/csi-driver/csi-kubevirt-hostpath-provisioner.yaml -n hostpath-provisioner
-    fi
-
-    # create StorageClass crc-csi-hostpath-provisioner
-    retry ${OC} apply -f kubevirt-hostpath-provisioner-csi/csi-sc.yaml
+function create_registry_pvc() {
 
     # Apply registry pvc with crc-csi-hostpath-provisioner StorageClass
     retry ${OC} apply -f registry_pvc.yaml
 
     # Add registry storage to pvc
     retry ${OC} patch config.imageregistry.operator.openshift.io/cluster --patch='[{"op": "add", "path": "/spec/storage/pvc", "value": {"claim": "crc-image-registry-storage"}}]' --type=json
+    # Chnage rollout method to Recreate, the default RollingUpdate is not supported with pvc accessmode ReadWriteOnce
```

Comment:
Why switch to readwriteonce in registry_pvc.yaml if it's causing issues?

Reply:
There's a "Chnage" typo

Reply:
> Why switch to readwriteonce in registry_pvc.yaml if it's causing issues?

`topolvm` doesn't support the `ReadWriteMany` access mode, and fails to create the volume with an error saying:

```
topolvm.io_lvms-operator-756689956-j2nxw_0a69a3a7-94d0-412d-9dbf-9f4b410aaece  failed to provision volume with StorageClass "lvms-rhel": rpc error: code = InvalidArgument desc = unsupported access mode: MULTI_NODE_MULTI_WRITER
```

but also `ReadWriteOnce` access mode is appropriate here, as [k8s docs](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes) says:

> ReadWriteMany
    the volume can be mounted as read-write by many nodes.

and we only have one node